central/main.go

```diff
@@ -820,6 +821,23 @@ func customRoutes() (customRoutes []routes.CustomRoute) {
 		},
 	)
 
+	repoMappingRoute := "/api/extensions/scanner-v4/repomappings"
+	utils.CrashOnError(err)
+	customRoutes = append(customRoutes,
+		routes.CustomRoute{
+			Route: repoMappingRoute,
+			Authorizer: perrpc.FromMap(map[authz.Authorizer][]string{
+				or.SensorOr(
+					or.ScannerOr(
+						user.With(permissions.View(resources.Administration)))): {
+					routes.RPCNameForHTTP(repoMappingRoute, http.MethodGet),
+				},
```

Comment:
I think Central also needs to get those mappings, at least for now, that we will be running the Indexer in the Central cluster.

Reply:
But the indexer will be calling this central endpoint, is that right?